lib/usd/ui/ItemDelegate.cpp

```diff
@@ -118,6 +118,7 @@ void ItemDelegate::setModelData(QWidget* editor, QAbstractItemModel* model, cons
 		if (nullptr != treeItem)
 		{
 			treeItem->setVariantSelectionModified();
+			Q_EMIT variantModified();
```

Comment:
I guess I could have cast the model pointer that is passed into this function to a TreeModel and told it directly to update its count (which is what this signal will end up triggering), but I was already hooking up signal/slots to make sure all the UI gets updated correctly so also doing this with a signal made sense.